src/api/playlistApi.tsx

```diff
@@ -0,0 +1,31 @@
+import { db } from '@/api/firebaseApp'
+import {
+  collection,
+  query,
+  orderBy,
+  limit,
+  startAfter,
+  getDocs
+} from 'firebase/firestore'
+
+const ITEMS_PER_PAGE = 10
+
+export const fetchPlaylists = async (lastVisible: boolean) => {
+  const playlistsRef = collection(db, 'Playlists')
+
+  let q = query(playlistsRef, orderBy('createdAt'), limit(ITEMS_PER_PAGE))
```

Comment:
P3;

위에 드린 리뷰랑 이어지는 건데요. 아래처럼 fetchLimit (변수명은 달라도 좋을 것 같아요)을 optional 인자로 두고, default 값을 지정해두되, 함수를 사용하는 개발자가 fetch할 갯수를 결정할 수 있게 개선하면 좋을 것 같습니다.
```suggestion
export const fetchPlaylists = async (lastVisible: boolean, fetchLimit= ITEMS_PER_PAGE) => {
  const playlistsRef = collection(db, 'Playlists')

  let q = query(playlistsRef, orderBy('createdAt'), limit(fetchLimit))
```